ark/utils/load_utils_test.py

```diff
@@ -96,15 +96,22 @@ def test_load_imgs_from_tree():
             load_utils.load_imgs_from_tree(temp_dir, img_sub_folder="TIFs", dtype="int16",
                                            channels=some_chans)
 
-        assert loaded_xr.equals(data_xr[:, :, :, :2], )
+        # need to reindex columns first before asserting equality
+        # cannot use xarray.equals for this because reindexing string coords changes their dtype
+        assert np.all(loaded_xr.values == data_xr[:, :, :, :2].loc[
```

Comment:
Why is this change necessary? We were sorting before and we're sorting now, so what's the difference?

Reply:
So the problem seems to happen when `channels` is unordered (ex. `['chan1', 'chan0']`). This can either happen because the argument to `channels` itself is unordered or becomes unordered when not all of the channels have a proper extension (in which `iou.list_files` is run, and that potentially can return an unordered list as well).

In this case, the `channel` coords in `loaded_xr` will likely be ordered differently than it would be in the test data (`data_xr`) created by `create_paired_xarray_fovs`. If we want to test that the corresponding values are the same, we need to reindex one of these arrays prior to testing equality because `.equals` in `xarray` will complain that the `channel` coords do not match between the two xarrays.

Reply:
Okay cool. In that case, maybe it would be worthwhile to add a few lines around 134 so that the channel names, with extensions, are in the same order as the supplied channels. I think it would be good for the channels, and fovs, to appear in the xarray in the order the user specified them.